### ISSUE TITLE
Remove file name truncation

### ISSUE THREAD

**Issue by davecan** [jdanielmourao/obsidian-sanctum#21]
Excellent theme, but I find myself constantly wrestling with the truncation of titles in the title bar. I use phrases as titles, some of which can be quite long, and this is a constant source of frustration when trying to rename the file because the phrase is cut off and it doesn't show me what I'm editing.

For example, I may have a title like this:

`Define desired outcomes and target dates instead of using casual handoffs, especially in remote work`

But it appears visually as:

![image](https://user-images.githubusercontent.com/121290/138878643-cff72242-aa39-401b-871b-a1a61b74ddfb.png)

Which in theory is fine, except that when I place the cursor in the title bar and move to the end of the title it appears like this:

![image](https://user-images.githubusercontent.com/121290/138878748-b10ed0d2-eaea-4290-8cff-a672d3bc9731.png)

I can edit/delete within that blank space but can't see the text I'm changing or typing.

What is the CSS rule used to perform this so I can create a snippet that will counteract it? It's unbelievably frustrating and introduces so much friction that I'm considering switching back away from the theme, which is otherwise excellent.

**Comment by davecan** [comment]
Hello sorry for the late response, got very busy and don't keep up with my email as much as I should.

I updated the theme (pressed the Update button in the theme viewer in Obsidian) and can confirm that at least for me it was NOT resolved. The behavior remained the same as reported.

However I did create a snippet with (only) this from your set of recommended snippets and can confirm it does at least hack around the problem:

```
.workspace-leaf.mod-active .view-header-title-container,
.view-header-title-container {
    height: fit-content;
}
```

It forces the title bar to change height to accommodate, which is not ideal but isn't unusable. It may also help me focus on reducing length of certain overly long titles. :)

Ideally though it will return to the previous Obsidian default which did not truncate. I've tried *many* themes and this was the first time I've encountered this particular issue.

Really nice theme overall beyond that issue though. Thanks for producing it.
